demos/src/demo-alternative.js

```diff
@@ -0,0 +1,28 @@
+// import './../../main.js';
```

Comment:
This file's possibly not needed if we use data attributes for the alternative demo :)